utils/bazel/llvm-project-overlay/clang-tools-extra/clangd/BUILD.bazel

```diff
@@ -0,0 +1,46 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+package(features = ["layering_check"],
```

Comment:
llvm/BUILD.bazel places `default_visibility` before `features`. Best to follow its convention.

Reply:
Done.